functests/1_performance/performance.go

```diff
@@ -116,6 +116,26 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			}
 		})
 
+		It("[test_id:32375][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] initramfs should not have injected configuration", func() {
+			for _, node := range workerRTNodes {
+				initramfsImagesPath, err := nodes.ExecCommandOnMachineConfigDaemon(&node, []string{"find", "/rootfs/boot/ostree/", "-name", "*.img"})
+				Expect(err).ToNot(HaveOccurred())
+				found := false
+				imagesPath := strings.Split(string(initramfsImagesPath), "\n")
+				for _, imagePath := range imagesPath[:2] {
+					initrd, err := nodes.ExecCommandOnMachineConfigDaemon(&node,
+						[]string{"lsinitrd", strings.TrimSpace(imagePath)})
+					Expect(err).ToNot(HaveOccurred())
+					initrdString := string(initrd)
+					if strings.Contains(initrdString, "'/etc/systemd/system.conf /etc/systemd/system.conf.d/setAffinity.conf'") {
+						found = true
+						break
```

Comment:
we break here once we found the string but the image is not necessarily the right one .
`find", "/rootfs/boot/ostree/", "-name", "*.img"`  can give several images of the image phases